Calculators/College/DebtCalculator.py

```diff
@@ -1,12 +1,12 @@
-debt = 95000
-interest_percent = .059
-yearly_salary = None
-hourly_salary = 20
-debt_repayment_percent = .2
-debt_repayment_length = 0
-work_days = 261
-work_hours = 8
-interest = 0
+debt = int(input('Debt: '))
```

Comment:
I had to manually converted the variables from string to int. You think we could make this conversion more efficient or you want to leave it as it is?